th2_common/schema/message/impl/rabbitmq/abstract_rabbit_sender.py

```diff
@@ -68,18 +75,17 @@ def send(self, message):
         content_counter = self.get_content_counter()
         content_counter.inc(self.extract_count_from(message))
 
-        cb = functools.partial(self.sending, self.channel, message)
-        self.connection.add_callback_threadsafe(cb)
-        self.connection.process_data_events()
+        for x in range(5):
```

Comment:
You don't need to check the channel status during every sending 
because you have the corresponding Exception for this.